testhelpers/docker_registry.go

```diff
@@ -1,59 +0,0 @@
-package testhelpers
-
-import (
-	"context"
-	"fmt"
-	"testing"
-	"time"
-
-	dockertypes "github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/container"
-	"github.com/docker/go-connections/nat"
-)
-
-type DockerRegistry struct {
```

Comment:
Just to be sure that I didn't miss something. You deleted this file because we don't use anywhere, right? Meaning, it doesn't relate to the analyzer acceptance tests, right?